### PR TITLE
Tg bot messaging impovement

### PR DESCRIPTION
**Pull request by esuwu**

### CODE REVIEW

File: cmd/tg_bot/internal/environment.go [hunk 1]
```diff
@@ -23,6 +25,19 @@ func (tgEnv *TelegramBotEnvironment) Start() {
 	log.Println("Telegram bot finished")
 }
 
+func (tgEnv *TelegramBotEnvironment) constructMessage(alert string, msg []byte) string {
+
+	message := `
```
Comment (nickeskov) [inline cmd/tg_bot/internal/environment.go:30]: Use http templates.

File: cmd/tg_bot/internal/environment.go [hunk 2]
```diff
@@ -40,7 +55,33 @@ func (tgEnv *TelegramBotEnvironment) SendMessage(msg []byte) {
...
+		messageToBot = tgEnv.constructMessage(entities.IncompleteAlertNotification, msg[1:])
+	case byte(entities.InvalidHeightAlertType):
+		messageToBot = tgEnv.constructMessage(entities.InvalidHeightAlertNotification, msg[1:])
+	case byte(entities.HeightAlertType):
+		messageToBot = tgEnv.constructMessage(entities.HeightAlertNotification, msg[1:])
+	case byte(entities.StateHashAlertType):
+		messageToBot = tgEnv.constructMessage(entities.StateHashAlertNotification, msg[1:])
+	case byte(entities.AlertFixedType):
+		messageToBot = tgEnv.constructMessage(entities.AlertFixedNotification, msg[1:])
+	}
```
Comment (nickeskov) [inline cmd/tg_bot/internal/environment.go:77]: Add default branch. You can simplify it by using map.

File: cmd/tg_bot/tg_bot.go [hunk 3]
```diff
@@ -38,7 +38,7 @@ func run() error {
 	flag.StringVar(&behavior, "behavior", "webhook", "Behavior is either webhook or polling")
 	flag.StringVar(&webhookLocalAddress, "webhook-local-address", ":8081", "The application's webhook address is :8081 by default")
 	flag.StringVar(&botToken, "bot-token", "", "Temporarily: the default token is the current token")
-	flag.StringVar(&publicURL, "public-url", "", "Default is https://mainnet-go-htz-fsn1-1.wavesnodes.com/bot")
+	flag.StringVar(&publicURL, "public-url", "", "")
```
Comment (nickeskov) [inline cmd/tg_bot/tg_bot.go:41]: Is this flag required?

File: cmd/tg_bot/internal/messages/base_messages.go [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+		"/start - the command to make the bot <b>start getting alerts</b>\n" +
+		"/mute -  the command to make the bot <b>stop listening to alerts</b>" +
+		"/help -  the command to see <b>information about bot</b> and available commands"
+
+	MuteText  = "Say no more..." + SleepingMsg
+	PongText  = "Pong!" + PongMsg
+	StartText = "Started monitoring..." + MonitoringMsg
+)
+
+//var baseCommands = []string{"ping", "hello", "start", "mute", "help"}
```
Comment (nickeskov) [inline cmd/tg_bot/internal/messages/base_messages.go:20]: Remove this unnecessary comment.

File: pkg/api/api.go [hunk 5]
```diff
@@ -27,7 +27,7 @@ func NewAPI(bind string, nodesStorage *nodes.Storage) (*API, error) {
 	r.Use(middleware.RealIP)
 	r.Use(middleware.Logger)
 	r.Use(middleware.Recoverer)
-	r.Use(middleware.SetHeader("Content-Type", "application/json"))
+	r.Use(middleware.SetHeader("Content-ShortDescription", "application/json"))
```
Comment (nickeskov) [inline pkg/api/api.go:30]: Revert it

File: pkg/messaging/client.go [hunk 6]
```diff
@@ -2,13 +2,24 @@ package messaging
...
 	_ "go.nanomsg.org/mangos/v3/transport/all"
+	"log"
+	"nodemon/pkg/entities"
 )
 
+func subscribeToAlerts(socket protocol.Socket) error {
+	for _, alertType := range entities.AlertTypes {
+		err := socket.SetOption(mangos.OptionSubscribe, []byte{byte(alertType)})
+		if err != nil {
+			return err
```
Comment (nickeskov) [inline pkg/messaging/client.go:17]: Add info about alert type.

**Review by nickeskov — CHANGES_REQUESTED** [review]
Also, sort imports, please.

File: cmd/tg_bot/tg_bot.go (hunk 3 above)
Comment (esuwu) [reply cmd/tg_bot/tg_bot.go:41]: Yes, it is. The telegram webhook requires a https url to send events

**Pull request closed without merging**